src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -141,28 +140,33 @@ public ShadowCompareRunsPane(ShadowController shadowController) {
         JLabel header = new JLabel("Comparison of PC2 vs. Remote Judgements");
         header.setAlignmentX(Component.CENTER_ALIGNMENT);
         this.add(header);
-        
+ 
+        this.setMinimumSize(size);shadowController.setFilter(FILTERS.NONE);//Required to reset the state of the filter
```

Comment:
Break this up into 2 lines. We don't like to put multiple statements on a single line.

Reply:
Solved